src/app/components/TicketCount.tsx

```diff
@@ -0,0 +1,26 @@
+import { FC } from 'react';
+
+type Props = {
+  ticketCount: number;
+  onClickMinus: () => void;
+  onClickPlus: () => void;
+};
+
+const TicketCount: FC<Props> = ({ ticketCount, onClickMinus, onClickPlus }) => {
```

Comment:
I like that you made this into it's own component, but I would separate concerns on another level. Here, this component basically only deals with rendering the UI, and takes no responsibility over interactions. I would give this component slightly more responsibility.

Instead of having two callbacks, for plus/minus, you could just have a single `onCountChange` callback that is called with the new value. You can also add a `max` property to allow this component to handle the clamping to a maximum value.

Compare with the basic `input` component in React/DOM. It has an `onChange` callback that is invoked whenever the value changes. As a user of the component, you don't have to worry about keyboard events, keeping track of where the text caret is, etc. The component deals with all of that for you and just lets you know when the value changes. That's a good ideal for a component like yours, as well.